scripts/dev/newReleaseBranch.py

```diff
@@ -0,0 +1,549 @@
+#######################################
+# This script creates HotFix branches for repo, share and packaging projects, and updates the master branches ready for the next SP/major release.
+# To use this script you need to install lxml:
+# 'pip install lxml'
+# Run the script without passing any arguments or with -h/--help argument to get usage information.
+# See below script behaviour explained.
+#######################################
+# Create a HotFix branches for the released version (for X.Y.Z release it will be X.Y.N eg., create 23.2.N for 23.2.0 release)
+# 1. acs-packaging:
+# - set RELEASE_VERSION to X.Y.Z+1, DEVELOPMENT_VERSION to X.Y.Z+1-SNAPSHOT in master_release.yml
+# - set POM versions to X.Y.Z+1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# 2. enterprise-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# 3. enterprise-repo:
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - set acs.version.label to .1
+# 4. community-share
+# - set scm-tag in main POM to HEAD
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set main POM versions to  X.Y.Z+1.1-SNAPSHOT
+# 5. community-repo:
+# - set ACS version properties in main POM to X.Y.Z+1
+# - set POM versions to  X.Y.Z+1.1-SNAPSHOT
+# - set scm-tag in main POM to HEAD
+# - increment schema by 1 in repository.properties?
```

Comment:
I think this is unnecessary for the HF release branch. If someone makes a patch for a HF then they will increment the schema for it.

Reply:
I didn't know how that was done practically. Historically we didn't bump that automatically when creating a branch. Should be easy to change, however, it is not harming us.

Reply:
I think it should be bumped for SP and LTS branches, but not HFs. You're right that it doesn't do any harm though.

Reply:
Not bumping that anymore.